addons/mail/static/src/components/message_list/message_list.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessageList" owl="1">
-        <div class="o_MessageList" t-att-class="{ 'o-empty': threadView and threadView.messages.length === 0, 'o-has-message-selection': props.selectedMessageLocalId }" t-on-scroll="onScroll">
+        <div class="o_MessageList" t-att-class="{ 'o-empty': threadView and threadView.messages.length === 0 }" t-on-scroll="onScroll">
```

Comment:
What happens to the css rule that was set on class `o-has-message-selection`?
Shouldn't condition on props `selectedMessageLocalId` be replaced by condition on `threadView.selectedMessage`?

Reply:
I found that this class is never used in CSS, so I went for removal as it seemed useless

Reply:
But it is used in message_list:
```scss
.o_MessageList_message.o-has-message-selection:not(.o-selected) {
    opacity: 0.5;
}
```

Reply:
It's indeed the same class name but on a different element, it's a message not the message list

![image](https://user-images.githubusercontent.com/48204996/99950761-58dd6b80-2d7d-11eb-8dfe-96b3f8428fcf.png)

Reply:
Damn that's confusing. Ok to remove it then.